interface/drugs/add_edit_drug.php

```diff
@@ -616,9 +616,9 @@ function sel_related() {
 
 </table>
 <div class="btn-group">
-    <input type='submit' class="btn btn-primary" name='form_save' value='<?php echo  $drug_id ? xla('Update') : xla('Add') ; ?>' />
+    <button type='submit' class="btn btn-primary btn-save" name='form_save' value='<?php echo  $drug_id ? xla('Update') : xla('Add') ; ?>'><?php echo  $drug_id ? xla('Update') : xla('Add') ; ?></button>
     <?php if (AclMain::aclCheckCore('admin', 'super') && $drug_id) { ?>
-    <input class="btn btn-danger" type='submit' name='form_delete' value='<?php echo xla('Delete'); ?>' />
+    <button class="btn btn-danger" type='submit' name='form_delete' value='<?php echo xla('Delete'); ?>'><?php echo xla('Delete'); ?></button>
```

Comment:
Now i'm curious! Does this work? Which gets passed on submit, name, value or both?
Normal syntax for a submit button is the value attr gets passed. Unsure how well parsers support using name to pass.

Reply:
When I tested this in the past (I also ran into same problem as Stu did), it appeared that both were passed with `form_delete` being the name of the parameter with the value set to `value`. So, if the form being submitted were a GET submission, it would look like:
add_edit_drug.php?form_delete=Delete

Reply:
Can test by by doing a print_r on the $_REQUEST (I'll plan to try this too in order to confirm my memory of things :) )